### PR TITLE
feat(portal): allow a user to display an API

### PR DESCRIPTION
**Pull request by phiz71**
Closes gravitee-io/issues#2789

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -119,6 +129,17 @@
     "error": "Une erreur est survenue pendant de le chargement de l'API",
     "subscribe": "Souscrire"
   },
+  "gv-info-api": {
+    "categories": "Catégories",
+    "global_information": "Informations générales",
+    "labels": "Tags",
+    "more_info": "Informations supplémentaires",
```
[the remark this answers is not in the document]
Comment (aelamrani) [reply src/assets/i18n/fr.json]: global => general?

File: src/app/pages/catalog/search/catalog-search.component.ts [hunk 2]
```diff
@@ -85,7 +85,7 @@ export class CatalogSearchComponent implements OnInit {
       .then((apisResponse: ApisResponse) => {
         if (apisResponse.data.length) {
           this.apiResults = apisResponse.data.map((a) => {
-            a.labels = this.apiLabelsPipe.transform(a);
+            // a.labels = this.apiLabelsPipe.transform(a);
```
[the remark this answers is not in the document]
Comment (phiz71) [reply src/app/pages/catalog/search/catalog-search.component.ts]: I need a Promise array, but `apisResponse.data` is an API array.

File: src/assets/i18n/fr.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (phiz71) [reply src/assets/i18n/fr.json]: I've changed more_info to moreInfo and deleted global_information

File: src/app/pages/api/api.component.ts [hunk 3]
```diff
@@ -14,31 +14,74 @@
...
 
   ngOnInit() {
     const apiId = this.route.snapshot.params.apiId;
     if (apiId) {
-      this.apiService.getPagesByApiId({ apiId, homepage: true })
+      this.apiServices.getPagesByApiId({ apiId, homepage: true })
         .subscribe(response => {
           this.homepage = response.data[0];
         });
+      this.currentApi = this.apiServices.getApiByApiId({ apiId }).toPromise();
```
[the remark this answers is not in the document]
Comment (phiz71) [reply src/app/pages/api/api.component.ts]: I've inlined instead

**Pull request merged**